sebastian-rueda/tarea-4/script.js

```diff
@@ -0,0 +1,138 @@
+class Calculator
```

Comment:
@SebastianRuedaGlobant , classes should be on their own file like /calculator.js